src/ert/config/ert_config.py

```diff
@@ -1427,7 +1427,7 @@ def _substitutions_from_dict(config_dict) -> dict[str, str]:
         subst_list[key] = val
```

Comment:
Small thing, but the variable name was left as `subst_list`, but it is no longer a list (list should probably never have been part of the name).